ko-search/movies-evan-likes/src/App/components/helpers/helpers.spec.js

```diff
@@ -0,0 +1,58 @@
+import test from 'tape';
```

Comment:
Tape doesn't have globals like mocha, and no need for beforeEach and afterEach related smells and shared state.

For dead simple unit testing I find tape to be pretty easy to read and use making it a more or less painless experience for unit testing.